Makefile

```diff
@@ -45,7 +45,7 @@ fmt: ## Reformats Terraform files accoring to standard
 	terraform fmt
 
 $(TFSEC): bin ## Installs tfsec
-	curl -L "$$(curl -s https://api.github.com/repos/liamg/tfsec/releases/latest | grep -o -E "https://.+?-$(OS)-amd64(.exe)?")" > $(TFSEC);\
+	curl -L "$$(curl -s https://api.github.com/repos/liamg/tfsec/releases/v0.21.0 | grep -o -E "https://.+?-$(OS)-amd64(.exe)?")" > $(TFSEC);\
```

Comment:
missing tags, should be `https://api.github.com/repos/liamg/tfsec/releases/tags/v0.21.0`